### PR TITLE
feature/analysis

### PR DESCRIPTION
**Pull request by william-stacken**
Should probably make use of jacoco

### CODE REVIEW

File: src/main/java/se/kth/iv1201/group4/recruitment/application/PersonService.java [hunk 1]
```diff
@@ -146,7 +146,10 @@ public UserDetails getLoggedInUser() {
                 && !(SecurityContextHolder.getContext().getAuthentication() instanceof AnonymousAuthenticationToken)) {
             return (UserDetails) auth.getPrincipal();
         } else {
-            LOGGER.info("Tried getting a logged in user, but none was available.");
+            LOGGER.debug("Tried getting a logged in user, but none was available.");
+            LOGGER.info("Logs info.");
+            LOGGER.debug("Logs debug.");
+            LOGGER.trace("Logs trace.");
```
Comment (fiilp) [inline src/main/java/se/kth/iv1201/group4/recruitment/application/PersonService.java]: I don't fully understand these 3 logs. Are they there for testing purposes?

Comment (william-stacken) [reply src/main/java/se/kth/iv1201/group4/recruitment/application/PersonService.java]: Those are leftovers from when I tried to get the logger to work. Spring apparently uses its own logger, so my log4j configuration wasn't even used, but it works now after excluding the right dependency. I will remove them.

**Review by Cactu5 — APPROVED** [review]
This looks good, but is it possible to get more detailed information from jacoco?

**Comment by william-stacken** [comment]
> This looks good, but is it possible to get more detailed information from jacoco?

You can go to the CI/CD job page and download an artifact that contains the full JaCoCo data

https://github.com/Cactu5/recruitment-application/actions/runs/593406852

**Pull request merged**